packages/zowe-explorer/README.md

```diff
@@ -66,6 +66,17 @@ Create a profile, review the sample use cases to familiarize yourself with the c
 
 You can now use all the functionalities of the extension.
 
+#### Profile Validation
+
+Zowe Explorer includes the profile validation feature that helps to ensure that z/OSMF is accessible and ready for use. If a profile is valid, the profile is active and can be used. By default, the feature is automatically enabled. You can disable the feature by right-clicking on your profile and selecting the **Disable Validation for Profile** option. Alternatively, you can enable or disable the feature for all profiles in the VS Code settings.
```

Comment:
We should use a more general term instead of "z/OSMF" here. Perhaps:

"helps to ensure that the specified connecction to z/OS is accessible and ready for use."

Reply:
Then I would suggest "helps to ensure that the specified connection to z/OS is successfully established and ready for use."

Reply:
Sounds good to me!